components/Router/Router.cjsx

```diff
@@ -51,6 +53,10 @@ component = ->
         <Route path="/PanelExamples" component={PanelExamples} />
 
         <Route path="/StandardListItemExamples" component={StandardListItemExamples} />
+
+        <Route path="/NavbarDropdownExamples" component={NavbarDropdownExamples}/>
```

Comment:
I think NavbarDropdownExamples should be removed from this PR because we are handling it with separate PR and it would create merge conflicts after we merge that other PR.